source/modules/MEI.gestural.xml

```diff
@@ -607,6 +569,30 @@
       <memberOf key="att.timestamp2.ges"/>
     </classes>
   </classSpec>
+  <classSpec ident="att.pitch.ges" module="MEI.gestural" type="atts">
+    <desc xml:lang="en">Gestural attributes about pitch.</desc>
+    <attList>
+      <attDef ident="oct.ges" usage="opt">
+        <desc xml:lang="en">Records performed octave information that differs from the written value.</desc>
+        <datatype>
+          <rng:ref name="data.OCTAVE"/>
+        </datatype>
+      </attDef>
+      <attDef ident="pname.ges" usage="opt">
+        <desc xml:lang="en">Contains a performed pitch name that differs from the written value.</desc>
+        <datatype>
+          <rng:ref name="data.PITCHNAME.GES"/>
+        </datatype>
+      </attDef>
+      <attDef ident="pnum" usage="opt">
```

Comment:
Shouldn't this be in the MIDI module? Something like `att.pitch.midi`?

Reply:
It's not just a MIDI number; could also be base-40 as per the description.

Reply:
I agree with @ahankinson, this seems to be broader than MIDI…

Reply:
Also agree